docker-images/oncoanalyser/Dockerfile

```diff
@@ -1,56 +1,75 @@
 FROM docker.io/continuumio/miniconda3:22.11.1
 
+# SOFTWARE VERSIONS
+ARG NEXTFLOW_VERSION="22.10.6"
+ARG DEBIAN_PLATFORM_VERSION="debian.11~bullseye_amd64"
+ARG CONTAINERD_IO_VERSION="1.6.18-1"
+ARG DOCKER_CE_VERSION="23.0.1-1"
+ARG DOCKER_CE_CLI_VERSION="23.0.1-1"
+ARG DOCKER_BUILDX_VERSION="0.10.2-1"
+ARG DOCKER_COMPOSE_VERSION="2.16.0-1"
+
+# ONCOANALYSER SOFTWARE PATH
+ARG ONCOANALYSER_GITHUB_REPO_URL="https://github.com/scwatts/oncoanalyser.git"
+ARG ONCOANALYSER_GITHUB_BRANCH_NAME="umccr"
+
+# Run apt updates
 RUN \
-  conda install conda-libmamba-solver
+  apt update -y -q
 
+# Set conda settings
 RUN \
+  conda install conda-libmamba-solver && \
   echo > ~/.condarc '\
 solver: libmamba\n\
 channels:\n\
   - conda-forge\n\
   - bioconda\n\
-  - defaults'
+  - defaults\n\
+'
 
+# Install conda deps
 RUN \
   conda install \
     jq \
-    'nextflow == 22.10.6' \
+    nextflow=="$NEXTFLOW_VERSION" \
     parallel \
     rclone \
     unzip
 
+# Install aws cli
 RUN \
-  wget -q 'https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip' && \
-    unzip awscli-exe-linux-x86_64.zip && \
+  wget --quiet 'https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip' \
+    --output-document "awscli.zip" && \
+    unzip -qq "awscli.zip" && \
     ./aws/install && \
-    rm -r aws/ awscli-exe-linux-x86_64.zip
-
-RUN \
-  wget -q -P /usr/bin/ 'https://stratus-documentation-us-east-1-public.s3.amazonaws.com/cli/1.2.0/linux/ica' && \
-  chmod 755 /usr/bin/ica
-COPY assets/ica_config.yaml /root/.ica/config.yaml
```

Comment:
Should also be safe to delete `assets/ica_config.yaml`.

Reply:
Resolved in f69c5dce51b421887ed895a06479ffe82c2428a3